src/common/src/hash/key.rs

```diff
@@ -150,7 +240,7 @@ pub trait HashKeySerDe<'a>: ScalarRef<'a> {
 /// group by implementation, but not join. In pg's join implementation, `null != null`, and the join
 /// executor should take care of this.
 pub trait HashKey:
-    EstimateSize + Clone + Debug + Hash + Eq + Sized + Send + Sync + 'static
+    EstimateSize + Clone + Debug + Hash + Eq + Sized + Send + Sync + 'static + GetBitmap
```

Comment:
Is it possible to directly put `null_bitmap` and `type Bitmap` to the `trait HashKey`?